server.js

```diff
@@ -32,61 +32,60 @@ app.use(bodyParser.json());
 app.use(express.static('public'));
 app.locals.title = 'BYOB';
 
-app.get('/', (request, response) => {
-});
-
 app.post('/api/v1/authenticate', (request, response) => {
   const { email, app_name } = request.body;
   const payload = { email, app_name };
   const authParams = ['email', 'app_name'];
 
-  for(let requiredParameter of authParams) {
-    if(!payload[requiredParameter]) {
+  for (let requiredParameter of authParams) {
+    if (!payload[requiredParameter]) {
       return response.status(422).send({
-        error: `Expected format: { email: <string>, app_name: <string> }. You are missing a "${requiredParameter}".`
-      })
+        error: `Expected format: { email: <string>, app_name: <string> }.
+           You are missing a "${requiredParameter}".`
```

Comment:
One of the tests failing Travis right now is this one. From looking at everything, my guess is that it's due to there being a "/n" - which is a nbsp (non-breaking space)? So I think we just need to make sure that the spacing in the server.js and routes.spec.js match up exactly.